src/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {
+  ListboxInput,
+  ListboxButton,
+  ListboxPopover,
+  ListboxList,
+  ListboxOption,
+} from '@reach/listbox';
+import '@reach/listbox/styles.css';
+import styled from 'styled-components';
+
+const StyledListboxInput = styled(ListboxInput)`
+  cursor: pointer;
+`;
+const StyledListboxPopover = styled(ListboxPopover)`
+  z-index: 200;
```

Comment:
You should really not pick random numbers. Do we have other z indices already? Otherwise would a `1` or `2` work already?

Reply:
yes, the header has 100. I was thinking of a global z-index file, maybe in another PR 🤔

Reply:
later through the theme is best. thanks